qiskit/pulse/instructions/instruction.py

```diff
@@ -44,7 +45,7 @@ def __init__(
         self,
         operands: Tuple,
         duration: int = None,
-        channels: Tuple[Channel] = None,
+        channels: Union[Tuple[Channel], Tuple[Frame]] = None,
```

Comment:
Seems like returning channels and frames under the name of `channels` is bit confusing to me.

Reply:
+1

Reply:
Is this relevant given the warning below?
```python
        if channels is not None:
            warnings.warn(
                "Specifying ``channels`` in the constructor is deprecated. "
                "All channels should be stored in ``operands``.",
                DeprecationWarning,
            )
```
I.e. it just seems that we should specify the Frame in the operands.